### PR TITLE
Fix Pagination in List can't change page size

### PR DESCRIPTION
**Pull request by ztplz**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. Describe the source of requirement, like related issue link.
https://github.com/ant-design/ant-design/issues/15405

2. Describe the problem and the scenario.

### 💡 Solution

1. How to fix the problem, and list final API implementation and usage sample if that is an new feature.

2. GIF or snapshot should be provided if includes UI/interactive modification.

### 📝 Changelog description

> Describe changes from userside, and list all potential break changes or other risks.

1. English description

2. Chinese description (optional)

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by ztplz** [comment]
https://github.com/react-component/pagination/blob/4b1c6b2c190ef15b84294c13224d4f36e17f0de4/src/Pagination.jsx#L2不能帮用户设置默认的pagesize  不然这里的if就进不去了

**Comment by zombieJ** [comment]
* CI failed
* Add an test case for this

**Comment by ztplz** [comment]
刚看了下  原来的测试根本就不对

**Comment by zombieJ** [comment]
> 刚看了下 原来的测试根本就不对

修一下哈~

**Comment by ztplz** [comment]
原来的测试好像没错  我的锅

**Comment by ztplz** [comment]
image
。。我本地跑没事

**Comment by zy410419243** [comment]
把 process.env.REACT 切成 15 跑应该就出错了

**Comment by ztplz** [comment]
image
这也行。。

File: components/list/__tests__/pagination.test.js [hunk 1]
```diff
@@ -109,6 +109,67 @@ describe('List.pagination', () => {
     expect(renderedNames(wrapper)).toEqual(['Jack', 'Lucy', 'Tom', 'Jerry']);
   });
 
+  it('should change pageSize via Select', () => {
+    if (process.env.REACT === '15') {
+      return;
+    }
```
Comment (afc163) [inline components/list/__tests__/pagination.test.js]: 最好在 React 15 中测试一下。

**Comment by ztplz** [comment]
at在15有毒

**Pull request closed without merging**